src/content/developers/docs/intro-to-ethereum/index.md

```diff
@@ -25,7 +25,7 @@ Watch Austin walk you through blockchains:
 
 <iframe width="100%" height="315" src="https://www.youtube.com/embed/zcX7OJ-L8XQ" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-If you want to see how blockchain hashes data, and then the previous block references all the past blocks, be sure to check out [this demo](https://andersbrownworth.com/blockchain/blockchain). It makes understanding the hashing of blocks really simple. 
+If you want to see how blockchain hashes data, and then the previous block references all the past blocks, be sure to check out [this demo](https://andersbrownworth.com/blockchain/blockchain). It makes understanding the hashing of blocks really simple.
```

Comment:
This is a confusing sentence. Maybe something like: "If you want to see how the blockchain hashes data, and how a block references all past blocks, be sure to check out [this demo]. It makes understanding the hashing of blocks really simple."

Reply:
this looks like an automatic linting change that removed a trailing space. similar below, unrelated to this PR